model-server-openapi/specifications/model-server-operative.yaml

```diff
@@ -15,18 +15,56 @@ info:
   version: "1.0.0"
 
 servers:
-  - url: '/'
+  - url: '/meta'
     description: model-server
 
 paths:
+  /version:
+    get:
+      operationId: getVersion
+      responses:
+        "200":
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Version"
   /metrics:
     get:
       operationId: getMetrics
       responses:
         "200":
           $ref: '#/components/responses/200'
 
+  /health:
+    get:
+      operationId: getHealth
+      responses:
+        "200":
+          description: healthy
+          content:
+            text/plain:
```

Comment:
We should make the responses (healthy and not healthy) an object to make it extensible.
Extensions to the health endpoint could be detailed statuses of used services.

https://opensource.zalando.com/restful-api-guidelines/#110

Reply:
Examples from Spring Boot are: https://docs.spring.io/spring-boot/docs/1.5.14.RELEASE/reference/html/production-ready-monitoring.html#production-ready-health-access-restrictions

Reply:
We actually haven't touched these much. There's also a ktor extension for realizing these endpoints which would simply bring a reasonable format.

Reply:
https://github.com/zensum/ktor-health-check

I think we talked about this extension in this context